.github/workflows/release.yaml

```diff
@@ -1,15 +1,17 @@
 name: "Release"
 
 on:
-  push:
-    branches:
-      - master
+  workflow_run:
+    workflows: "Test"
```

Comment:
My only hesitation in merging this before fixing [these flaky tests](https://github.com/FuelLabs/fuels-ts/issues/1755) is that it can potentially halt subsequent releases _(we should do one today)_, which may be a bit paradoxical because blocking releases _is_ what we want. 

However, flaky tests are not what we're proofing against, and it wouldn't be fun to have to battle with this for this next release specifically.

Thoughts?

Reply:
That's a dilemma. Let's do a coin toss to decide 😆 jk I would say let's wait for the flaky tests to get resolved, and then merge this in. Let's not introduce anything that could potentially delay today's release

Reply:
I am pro us introducing this if it means we resolve issues (such as flaky tests) with a higher level of urgency, as they could be indicative of a different problem.

Reply:
@danielbate It's best to resolve the [flakiness](https://github.com/FuelLabs/fuels-ts/issues/1755) with a higher level of urgency _without_ necessarily halting our release process, which can create an additional layer of problems. 🙂